### PR TITLE
disable report for local csp

### PR DESCRIPTION
**Pull request by abhinavkgrd**
## Description

^ title

## Test Plan

tested locally

### CODE REVIEW

File: configUtil.js [hunk 1]
```diff
@@ -26,8 +26,6 @@ module.exports = {
         'base-uri ': "'self'",
         'frame-ancestors': " 'none'",
         'form-action': "'none'",
-        'report-uri': 'https://csp-reporter.ente.io',
```
Comment (vishnukvmd) [inline configUtil.js:29]: This doesn't seem specific to local?

Comment (abhinavkgrd) [reply configUtil.js:29]: these configurations are only for the local development server.
we were sending local reports to the same CSP-reporter webhook.
disabling that. as they are just adding noise to that group

Comment (abhinavkgrd) [reply configUtil.js:29]: for production configs we have to _header file in /public folder

Comment (abhinavkgrd) [reply configUtil.js:29]: @vishnukvmd

**Review by vishnukvmd — APPROVED** [review]

**Pull request merged**